osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -211,7 +211,7 @@ public class FilterableDifficultyIcon : DifficultyIcon
         {
             private readonly BindableBool filtered = new BindableBool();
 
-            private readonly CarouselBeatmap item;
+            public readonly CarouselBeatmap Item;
```

Comment:
Not too sure what to feel about this - ideally this exposition as `public` shouldn't be necessary, but it might be somewhat difficult to test without it (would most likely have to depend on beatmap item order in the carousel matching the icon order).